src/Microsoft.PowerApps.TestEngine/TestEngineEventHandler.cs

```diff
@@ -17,14 +17,14 @@ public class TestEngineEventHandler : ITestEngineEvents
 
         // NOTE: Any changes to these messages need to be handled in the consuming tool's console event handler, like in pac cli tool.
         // These console messages need to be considered for localization.
-        public static string UserInputExceptionInvalidTestSettingsMessage = "   Invalid test settings specified in testconfig. For more details, check the logs.";
+        public static string UserAppExceptionMessage = "   [Critical Error] Could not access PowerApps. For more details, check the logs.";
         public static string UserInputExceptionInvalidFilePathMessage = "   Invalid file path. For more details, check the logs.";
+        public static string UserInputExceptionInvalidOutputPathMessage = "Invalid output directory; does not exist. For more details, check the logs.";
```

Comment:
Would be nice to test the newly added console message here 
src/Microsoft.PowerApps.TestEngine.Tests/ConsoleOutputTests.cs

Reply:
Done! :)